staff/monica-morales/layouts/flex.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Flex</title>
+</head>
+<style>
+    * {
+        box-sizing: border-box;
+    }
+
+    body {
+        margin: 0;
+    }
+
+    .container {
+        display: flex;
+        height: 100vh;
+        flex-direction: column;
+    }
+
+    .header {
+        height: 10vh;
+        width: 100%;
+        border: 2px solid black;
+    }
+
+    .nav {
+        height: 100%;
+        width: 100%;
+        border: 2px solid black;
+    }
+
+    .main {
+        height: 100%;
+        width: 100%;
+        border: 2px solid black;
+        padding-left: 5px;
+    }
+
+    .aside {
+        height: 100%;
+        width: 100%;
+        border: 2px solid black;
+    }
+
+    .footer {
+        height: 10vh;
+        width: 100%;
+        border: 2px solid black;
+        flex-direction: column;
+    }
+
+    .container >* {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    .container > div {
+        display: flex;
+        height: 80vh;
+        flex-direction: column;
+    }
+
+    .nav, .main, .aside {
+        display:flex;
+        justify-content: center;
+        align-items: center;
+    }
+  
+    @media only screen and (min-width: 992px) {
```

Comment:
Use standard sizes for the media queries like
320px.
480px.
600px.
768px.
900px.
1024px.
1200px.

Reply:
Thanks Dario, I will consider it for future occasions